ntirpc/rpc/svc.h

```diff
@@ -505,27 +517,41 @@ static inline void svc_release_it(SVCXPRT *xprt, u_int flags,
 #define SVC_RELEASE(xprt, flags)					\
 	svc_release_it(xprt, flags, __func__, __LINE__)
 
+#define SVC_DESTROY_RETRY 10
 /* SVC_DESTROY() is SVC_RELEASE() with once-only semantics.
  * Idempotent SVC_XPRT_FLAG_DESTROYED (bit SVC_XPRT_FLAG_DESTROYING)
  * indicates that more references should not be taken.
  */
 static inline void svc_destroy_it(SVCXPRT *xprt,
 				  const char *tag, const int line)
 {
+	int retry;
 	uint16_t flags = atomic_postset_uint16_t_bits(&xprt->xp_flags,
 						      SVC_XPRT_FLAG_DESTROYING);
 
 	XPRT_TRACE(xprt, __func__, tag, line);
 
-#ifdef USE_LTTNG_NTIRPC
-	tracepoint(xprt, destroy, tag, line, xprt, flags);
-#endif /* USE_LTTNG_NTIRPC */
+	XPRT_AUTO_TRACEPOINT(xprt, destroy_it, TRACE_DEBUG, "Destroy XPRT");
 
 	if (flags & SVC_XPRT_FLAG_DESTROYING) {
 		/* previously set, do nothing */
 		return;
 	}
 
+	/* A newly created xprt will be 1. inserted into the tree
```

Comment:
This chunk is not part of LTTng changes.

Reply:
Looks like I had some rebase issues. Fixed all issues I could find.